src/Tribe/RSVP.php

```diff
@@ -2016,76 +2461,42 @@ public function generate_tickets_for( $product_id, $ticket_qty, $attendee_detail
 		 */
 		do_action( 'tribe_tickets_rsvp_before_attendee_ticket_creation', $post_id, $ticket_type, $_POST );
 
+		$attendee_ids = [];
+
 		// Iterate over all the amount of tickets purchased (for this product)
 		for ( $i = 0; $i < $qty; $i++ ) {
-
-			$attendee           = array(
-				'post_status' => 'publish',
-				'post_title'  => $attendee_full_name . ' | ' . ( $i + 1 ),
-				'post_type'   => self::ATTENDEE_OBJECT,
-				'ping_status' => 'closed',
-			);
-
-			// Insert individual ticket purchased
-			$attendee_id = wp_insert_post( $attendee );
-
-			if ( $status_stock_size > 0 ) {
-				// adjust total sales
-				$sales = (int) get_post_meta( $product_id, 'total_sales', true );
-				update_post_meta( $product_id, 'total_sales', ++ $sales );
-
-				// adjust stock
-				$stock = (int) get_post_meta( $product_id, '_stock', true ) - $status_stock_size;
-				update_post_meta( $product_id, '_stock', $stock );
+			try {
+				$attendee_ids[] = $this->create_attendee_for_ticket( $ticket_type, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 8bac0a19d030e39ac0e3eba1210d38a8</sub>